### PR TITLE
IGVF-332 Added placeholder for CCF uuid

### PR DESCRIPTION
**Pull request by KAndreeva**
Added placeholder for CCF uuid.

### CODE REVIEW

File: src/igvfd/schemas/tissue.json [hunk 1]
```diff
@@ -77,6 +88,14 @@
                 "week"
             ]
         },
+        "ccf_id": {
+            "title": "Common coordinate framework identifier",
+            "description": "This identifier associates an entry from the HubMap Consortium for specimens, biological structures, and spatial positions.",
```
Comment (gabdank) [inline src/igvfd/schemas/tissue.json:93]: can we please change description to : "HubMap Common Coordinate Framework unique identifier corresponding to the organ, biological structure, and spatial location of the tissue specimen."

File: src/igvfd/schemas/tissue.json [hunk 2]
```diff
@@ -77,6 +88,14 @@
...
+            "comment": "This identifier will be later linked to: https://portal.hubmapconsortium.org/ccf-eui",
+            "type": "string",
+            "format": "uuid",
+            "nullable": true
```
Comment (gabdank) [inline src/igvfd/schemas/tissue.json:97]: why should it be "nullable" true?

Comment (gabdank) [reply src/igvfd/schemas/tissue.json:97]: either we have or we don't, i don't think we need nullable

File: src/igvfd/tests/data/inserts/tissue.json [hunk 3]
```diff
@@ -110,6 +112,7 @@
         "part_of": "IGVFSM000KAS",
         "source": "/labs/j-michael-cherry/",
         "nih_institutional_certification": "NICHD1442",
+        "ccf_id": null,
```
Comment (gabdank) [inline src/igvfd/tests/data/inserts/tissue.json:115]: i don't think we need this

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/igvfd/schemas/tissue.json (hunk 1 above)
Comment (KAndreeva) [reply src/igvfd/schemas/tissue.json]: Done

File: src/igvfd/schemas/tissue.json (hunk 2 above)
Comment (KAndreeva) [reply src/igvfd/schemas/tissue.json]: done

**Review by gabdank — APPROVED** [review]

**Pull request merged**